.npmignore

```diff
@@ -0,0 +1,5 @@
+.npm
+.husky
+.commitlintrc
+release.config.js
+renovate.json
```

Comment:
not sure about this, probably we need it, but we need to setup CI steps for it I guess

Reply:
Not sure I understand sorry. I think this config is consumed by renovate github app, do I understand this correctly ?

Reply:
It seems that it is needed for internal gitlab as you mentioned below.
Is there are any particular reason we don't use it for our open source projects as well? 
seems like in the docs there is a section about GH as well?

Reply:
😄 sorry, I was talking about .npmignore file. Speaking about renovate bot, our OSS is in very early stage and it doesn't have such developed infrastructure atm. But it worth to check renovate bot and other alternatives that we could use in github